### PR TITLE
Move relevant pages from the current wiki

### LINKED ISSUE #6: Move relevant pages from the current wiki
# Problem
The new documentation is missing content from certain pages and only includes references to them. This can be seen in the following example (running a node):

- Deployed page URL: https://wiki.sora.org/guides/staking/how-to-run-a-sora-mainnet-node
- Markdown file for the current wiki page: https://github.com/sora-xor/wiki/blob/master/guides/staking/how-to-run-a-sora-mainnet-node.md
- Location of the same file in the new wiki repository: https://github.com/sora-xor/sora-docs/blob/develop/src/running-a-node.md

Copying and pasting the markdown file results in an error, as shown in the following figure:

Screenshot 2023-04-21 at 12 27 22

# Resolution
To resolve this issue, manual editing of the markdown files is necessary to ensure compatibility with the new wiki platform, vitepress.dev.

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

File: src/nominating-validators.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Nominating Validators
+
+Nominating validators is an important part of the SORA ecosystem as it helps to secure and decentralize the network. While you can nominate any validator, you should do research to make sure the validators you nominate are high quality and won't attack the network. If validators cause harm to the network, you will lose money as a nominator!
```
Comment (WRRicht3r) [inline src/nominating-validators.md]:
```suggestion
Nominating validators is an important part of the SORA ecosystem as it helps to secure and decentralize the network. While you can nominate any validator, you should do your own research to make sure the validators you nominate are high quality and won't attack the network. If validators cause harm to the network, you will lose money as a nominator!
```

File: src/nominating-validators.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- cnScvgXFLQgeNFoVfMjf2U5MC24CkdRkLTUKbJYefYRwqy3VZ
+- cnSgH7nHuiPk5Z2cubwsFZGdmzvzhrgBKgTqLCeR7hyo6dek8
+- cnU2w3CPn9narRKeSamHmKBBaSXY949A4udq6APBXNkmnkwsJ
+- cnV58Z8zT37K6iUsijRjYHBzk95aTqvLfn3WvvfJcCwFQKHCL
+- cnRrijYYHb4fWPuF9MraEvm8nvfztYDxcAZfKfN2AyKLBn81A
+- cnUDwD4nPorodyd3inwThA2yBt2dFuHTfnvKowH6dirk47Qqr
+
+To nominate a Validator successfully on the SORA network, you need:
+
+- [Polkadot.js extension](https://polkadot.js.org/extension/) installed on your browser, with an account created and some XOR (for details, please see the first tutorial of this section)
```
Comment (WRRicht3r) [inline src/nominating-validators.md]:
```suggestion
- The [Polkadot.js extension](https://polkadot.js.org/extension/) installed on your browser, with an account created and some XOR 
```

File: src/nominating-validators.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+Funding operations are controlled by the stash and any other non-funding actions by the controller itself. This operation increases the security of your funds.
+
+You can read more [here](https://www.holdpolkadot.com/post/stash-account-controller-account-on-polkadot-and-kusama).
+
+4. Set the amount you want to stake and click on "Next"
+
+5. The next step is to choose the Validators you want to nominate.
+
+SORA Network uses [Phragmèn Method](https://wiki.polkadot.network/docs/en/learn-phragmen) to select Validators.
```
Comment (WRRicht3r) [inline src/nominating-validators.md]:
```suggestion
SORA Network uses the [Phragmèn Method](https://wiki.polkadot.network/docs/en/learn-phragmen) to select Validators.
```

File: src/nominating-validators.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+During each era, Validators are chosen using the Phragmen Method, and those who nominated them will receive a reward (a stake reward).
+
+From a Nominator's perspective, you only need to nominate at least one of the Validators who is actually elected during each era. This explains why you can nominate more than one candidate at the same time, up to 16
```
Comment (WRRicht3r) [inline src/nominating-validators.md]:
```suggestion
From a Nominator's perspective, you only need to nominate at least one of the Validators who is actually elected during each era. This explains why you can nominate more than one candidate at the same time, with a maximum of 16 validators per nominator account.
```

File: src/polkaswap-explore.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Explore
+
+The ["Explore" page](https://polkaswap.io/#/explore/demeter/farming) in Polkaswap.io is designed to help users discover and participate in decentralized finance activities such as pooling and staking, as well as discover information on the tokens currently available. It has several tabs that provide access to different sections of the platform. Here is a breakdown of each tab:
+
+[Farming:](https://polkaswap.io/#/explore/demeter/farming) This tab displays a list of available farming opportunities on Polkaswap. Users can stake their LP tokens in the designated pools to earn rewards in the form of PSWAP tokens.
```
Comment (WRRicht3r) [inline src/polkaswap-explore.md]:
```suggestion
- [Farming:](https://polkaswap.io/#/explore/demeter/farming) This tab displays a list of available farming opportunities on Polkaswap. Users can stake their LP tokens in the designated pools to earn rewards in the form of PSWAP tokens.
```

File: src/polkaswap-explore.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+![](.gitbook/assets/polkaswap-explore-farming.png)
+
+[Pools:](https://polkaswap.io/#/explore/pools/) The tab displays a list of available pools along with their respective APRs and total liquidity. The Pools tab helps users be informed to provide liquidity to various trading pairs on Polkaswap. By supplying liquidity, users can earn trading fees and other rewards.&#x20;
```
Comment (WRRicht3r) [inline src/polkaswap-explore.md]:
```suggestion
- [Pools:](https://polkaswap.io/#/explore/pools/) The tab displays a list of available pools along with their respective APRs and total liquidity. The Pools tab helps users be informed to provide liquidity to various trading pairs on Polkaswap. By supplying liquidity, users can earn trading fees and other rewards.
```

File: src/polkaswap-explore.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+![](.gitbook/assets/polkaswap-explore-pools.png)
+
+[Staking:](https://polkaswap.io/#/explore/demeter/staking) The tab displays a list of available staking options on Polkaswap.io along with their respective APRs and total locked liquidity. Users can stake their tokens in the designated pools and earn rewards.&#x20;
```
Comment (WRRicht3r) [inline src/polkaswap-explore.md]:
```suggestion
- [Staking:](https://polkaswap.io/#/explore/demeter/staking) The tab displays a list of available staking options on Polkaswap.io along with their respective APRs and total locked liquidity. Users can stake their tokens in the designated pools and earn rewards.
```

File: src/polkaswap-explore.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+![](.gitbook/assets/polkaswap-explore-staking.png)
+
+[Tokens:](https://polkaswap.io/#/explore/tokens) The Tokens tab displays a list of all available tokens on the Polkaswap platform and provides detailed information about each token, including its current price, total value, token velocity, and trading volume. Users can filter the list by different criteria such as name, symbol, and trading volume.
```
Comment (WRRicht3r) [inline src/polkaswap-explore.md]:
```suggestion
- [Tokens:](https://polkaswap.io/#/explore/tokens) The Tokens tab displays a list of all available tokens on the Polkaswap platform and provides detailed information about each token, including its current price, total value, token velocity, and trading volume. Users can filter the list by different criteria such as name, symbol, and trading volume.
```

File: src/polkaswap-other-features.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+# Other Features
+
+On the top right corner of the Polkaswap page, you'll find a few additional features that can enhance your experience.&#x20;
```
Comment (WRRicht3r) [inline src/polkaswap-other-features.md]:
```suggestion
On the top right corner of the Polkaswap page, you'll find a few additional features that can enhance your experience.
```

File: src/polkaswap-statistics.md [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+
+The ["Statistics" page](https://polkaswap.io/#/stats) provides important information about the transactions, accounts, fees, and token supply on Polkaswap.io. It includes charts and data for different timeframes, allowing users to analyze the platform's performance over time.
+
+![](.gitbook/assets/polkaswap-statistics.png)
+
+Key metrics such as TVL (Total Value Locked) and overall trading Volume are displayed, as well as the percentage increase or decrease in various categories. Users can also view the transaction activity between SORA and Ethereum networks (HASHI bridge).
+
+![](.gitbook/assets/polkaswap-statistics-fees-and-supply.png)
+
+The ["Statistics" page](https://polkaswap.io/#/stats) is a valuable tool for anyone looking to track the performance of Polkaswap.
```
Comment (WRRicht3r) [inline src/polkaswap-statistics.md]:
```suggestion
The ["Statistics" page](https://polkaswap.io/#/stats) is a valuable tool for anyone looking to track the performance of the tokens on Polkaswap.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by WRRicht3r — APPROVED** [review]

File: src/polkaswap-other-features.md [hunk 11]
```diff
@@ -0,0 +1,9 @@
+# Other Features
+
+On the top right corner of the Polkaswap page, you'll find a few additional features that can enhance your experience.
```
Comment (outoftardis) [inline src/polkaswap-other-features.md]:
```suggestion
In the top right corner of the Polkaswap page, you'll find a few additional features that can enhance your experience.
```

File: src/polkaswap-other-features.md [hunk 12]
```diff
@@ -0,0 +1,9 @@
...
+
+These features include the ability to switch to dark or light mode, select a language from a list of available options, enable browser notifications for balance changes and other updates, as well as hide your balances for privacy. \
+
+![](.gitbook/assets/polkaswap-other-features.png)
+
+You can easily access these features by clicking on the corresponding icons on the top right corner of the page.
```
Comment (outoftardis) [inline src/polkaswap-other-features.md]:
```suggestion
You can easily access these features by clicking on the corresponding icons in the top right corner of the page.
```

File: src/nominating-validators.md [hunk 13]
```diff
@@ -0,0 +1,65 @@
...
+- cnU2w3CPn9narRKeSamHmKBBaSXY949A4udq6APBXNkmnkwsJ
+- cnV58Z8zT37K6iUsijRjYHBzk95aTqvLfn3WvvfJcCwFQKHCL
+- cnRrijYYHb4fWPuF9MraEvm8nvfztYDxcAZfKfN2AyKLBn81A
+- cnUDwD4nPorodyd3inwThA2yBt2dFuHTfnvKowH6dirk47Qqr
+
+To nominate a Validator successfully on the SORA network, you need:
+
+- The [Polkadot.js extension](https://polkadot.js.org/extension/) installed on your browser, with an account created and some XOR
+
+NOTE: Staking rewards **expire** after 84 eras. On **SORA**, that's about 21 days.
```
Comment (outoftardis) [inline src/nominating-validators.md]:
```suggestion
::: info 
Staking rewards **expire** after 84 eras. On **SORA**, that's about 21 days.
:::
```

File: src/nominating-validators.md [hunk 14]
```diff
@@ -0,0 +1,65 @@
...
+
+1. Go to SORA[ polkadot.js](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fws.sora2.soramitsu.co.jp#/staking), click on "Network" and then "Staking". **Make sure you are connected to the SORA Network**, you should see the logo at the top left.
+
+![](.gitbook/assets/polkadotjs-staking-tab.png)
+
+2. Next, you will need to create a new Nominator Stash
+
+Click on "Account Sections" and then "Nominator", as shown in the pictures below.
+
+Make sure to read the information displayed carefully.
```
Comment (outoftardis) [inline src/nominating-validators.md]:
```suggestion
2. Next, you will need to create a new Nominator Stash. Click on "Account Sections" and then "Nominator", as shown in the pictures below. Make sure to read the information displayed carefully.
```

File: src/nominating-validators.md [hunk 15]
```diff
@@ -0,0 +1,65 @@
...
+
+![](.gitbook/assets/polkadotjs-nominator-button.png)
+
+![](.gitbook/assets/polkadotjs-setup-nominator.png)
+
+3. Set the Stash and Controller accounts.
+
+Think of the stash as your cold wallet and the controller as your hot wallet.
+
+Funding operations are controlled by the stash and any other non-funding actions by the controller itself. This operation increases the security of your funds.
```
Comment (outoftardis) [inline src/nominating-validators.md]:
```suggestion
Funding operations are controlled by the stash, while non-funding actions are managed by the controller itself. Having the stash and the controller accounts makes your funds more secure.
```

File: src/nominating-validators.md [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+
+You can read more [here](https://www.holdpolkadot.com/post/stash-account-controller-account-on-polkadot-and-kusama).
+
+4. Set the amount you want to stake and click on "Next"
```
Comment (outoftardis) [inline src/nominating-validators.md]:
```suggestion
4. Set the amount you want to stake and click "Next".
```

File: src/nominating-validators.md [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+
+5. The next step is to choose the Validators you want to nominate.
+
+SORA Network uses the [Phragmèn Method](https://wiki.polkadot.network/docs/en/learn-phragmen) to select Validators.
+
+During each era, Validators are chosen using the Phragmen Method, and those who nominated them will receive a reward (a stake reward).
+
+From a Nominator's perspective, you only need to nominate at least one of the Validators who is actually elected during each era. This explains why you can nominate more than one candidate at the same time, with a maximum of 16 validators per nominator account.
+
+**Disclaimer:** it is important to nominate trustworthy and reliable validators. Validators and their Nominators' stake can be slashed, which is a punishment for failing to follow the protocol. You can read more about Slashing\* [here](https://w3f-research.readthedocs.io/en/latest/polkadot/slashing/amounts.html)
```
Comment (outoftardis) [inline src/nominating-validators.md]: maybe use the `:::` syntax for this disclaimer

I think this should work (pls check the output though)

```
::: info Disclaimer

:::
```

File: src/polkaswap-connect-wallet.md [hunk 18]
```diff
@@ -0,0 +1,20 @@
+# Connect Wallet
+
+1.  Go to [Polkaswap.io](https://polkaswap.io) and click on the "Connect account" button at the top of the page.\
```
Comment (outoftardis) [inline src/polkaswap-connect-wallet.md]:
```suggestion
1. Go to [Polkaswap.io](https://polkaswap.io) and click on the "Connect account" button at the top of the page.
```

File: src/polkaswap-connect-wallet.md [hunk 19]
```diff
@@ -0,0 +1,20 @@
...
+
+![](.gitbook/assets/connect-wallet.png)
+
+2.  You will see a list of supported wallet options to connect, including [Fearless Wallet](https://fearlesswallet.io), Polkadot.js, Subwallet, and Talisman.\
```
Comment (outoftardis) [inline src/polkaswap-connect-wallet.md]:
```suggestion
2. You will see a list of supported wallet options to connect, including [Fearless Wallet](https://fearlesswallet.io), Polkadot.js, Subwallet, and Talisman.
```

File: src/polkaswap-connect-wallet.md [hunk 20]
```diff
@@ -0,0 +1,20 @@
...
+
+![](.gitbook/assets/choose-wallet-to-connect.png)
+
+3.  Select the wallet extension installed on your browser. For example, if you are using Fearless Wallet, click on the Fearless Wallet option
```
Comment (outoftardis) [inline src/polkaswap-connect-wallet.md]:
```suggestion
3. Select the wallet extension installed on your browser. For example, if you are using Fearless Wallet, click on the Fearless Wallet option.
```

File: src/polkaswap-connect-wallet.md [hunk 21]
```diff
@@ -0,0 +1,20 @@
...
+4.  A pop-up window will appear from your selected wallet extension, asking you to confirm the connection to Polkaswap. Click "Allow" or "Confirm" to proceed.
```
Comment (outoftardis) [inline src/polkaswap-connect-wallet.md]:
```suggestion
4. A pop-up window will appear from your selected wallet extension, asking you to confirm the connection to Polkaswap. Click "Allow" or "Confirm" to proceed.
```
why is it `or` in `"Allow" or "Confirm"`?

File: src/polkaswap-connect-wallet.md [hunk 22]
```diff
@@ -0,0 +1,20 @@
...
+5.  After the connection has been established, you will see a list of your available accounts displayed on the Polkaswap page. Choose the account you want to use and click on it\
```
Comment (outoftardis) [inline src/polkaswap-connect-wallet.md]:
```suggestion
5. After the connection has been established, you will see a list of your available accounts displayed on the Polkaswap page. Choose the account you want to use and click on it.
```

File: src/polkaswap-connect-wallet.md [hunk 23]
```diff
@@ -0,0 +1,20 @@
...
+
+![](.gitbook/assets/select-an-account.png)
+
+6.  Once the connection is complete, you will be able to see your account balance and manage your funds on Polkaswap.
```
Comment (outoftardis) [inline src/polkaswap-connect-wallet.md]:
```suggestion
6. Once the connection is complete, you will be able to see your account balance and manage your funds on Polkaswap.
```

File: src/polkaswap-connect-wallet.md [hunk 24]
```diff
@@ -0,0 +1,20 @@
...
+7.  Congratulations, you have successfully connected your wallet! Now you can stake, provide liquidity, trade, and much more with style and freedom on Polkaswap!\
```
Comment (outoftardis) [inline src/polkaswap-connect-wallet.md]:
```suggestion
Congratulations, you have successfully connected your wallet! Now you can stake, provide liquidity, trade, and much more with style and freedom on Polkaswap!
```

File: src/polkaswap-connect-wallet.md [hunk 25]
```diff
@@ -0,0 +1,20 @@
...
+    \
```
Comment (outoftardis) [inline src/polkaswap-connect-wallet.md]:
```suggestion
```

File: src/polkaswap-connect-wallet.md [hunk 26]
```diff
@@ -0,0 +1,20 @@
...
+    Remember to always do your own research before making any financial decisions.
```
Comment (outoftardis) [inline src/polkaswap-connect-wallet.md]:
```suggestion
Remember to always do your own research before making any financial decisions.
```

File: src/polkaswap-send-and-receive.md [hunk 27]
```diff
@@ -0,0 +1,41 @@
...
+4.  Paste the receiver's address in the "Address" field
+
+![](.gitbook/assets/polkaswap-send-address.png)
+
+5.  Enter the amount you want to send in the "Amount" field.
+6.  Review the network fee for the transaction. Please note that you need to have XOR for fees on your transferrable balance for the transaction to be successful
+7.  Review the transaction details to ensure everything is correct, including the receiver's address and the amount of tokens being sent.
+8.  Confirm the transaction and sign it using your wallet extension.
+
+    Done! Your transaction is now complete!\
```
Comment (outoftardis) [inline src/polkaswap-send-and-receive.md]:
```suggestion
1. Go to the [Account tab](https://polkaswap.io/#/wallet) on Polkaswap.

![](.gitbook/assets/account-tab.png)

2. Choose the asset you want to send from the list of available assets. If you don't see the asset you want to send, click on the "add asset" button and add it to the list.
3. Click on the "Send" button next to the asset you want to send.

![](.gitbook/assets/polkaswap-send.png)

4. Paste the receiver's address in the "Address" field.

![](.gitbook/assets/polkaswap-send-address.png)

5. Enter the amount you want to send in the "Amount" field.
6. Review the network fee for the transaction. Please note that you need to have XOR for fees on your transferrable balance for the transaction to be successful.
7. Review the transaction details to ensure everything is correct, including the receiver's address and the amount of tokens being sent.
8. Confirm the transaction and sign it using your wallet extension.

Done! Your transaction is now complete!
```

File: src/polkaswap-send-and-receive.md [hunk 28]
```diff
@@ -0,0 +1,41 @@
...
+
+![](.gitbook/assets/polkaswap-receive-qr-code-button.png)
+
+3.  Select the asset you would like to receive.
+4.  You will be redirected to a page with your wallet address in both text and QR code format.\
+
+![](.gitbook/assets/polkaswap-receive-qr-code.png)
+
+5.  Copy your wallet address or share the QR code with the sender.
+6.  The sender can then use your wallet address or scan the QR code to send you the desired asset.
```
Comment (outoftardis) [inline src/polkaswap-send-and-receive.md]:
```suggestion
1. Go to the [Account tab](https://polkaswap.io/#/wallet) on Polkaswap.

![](.gitbook/assets/account-tab.png)

2. Choose the asset you'd like to receive and click on the "Receive" button.

![](.gitbook/assets/polkaswap-receive-qr-code-button.png)

3. Select the asset you would like to receive. You will be redirected to a page with your wallet address in both text and QR code format.

![](.gitbook/assets/polkaswap-receive-qr-code.png)

5. Copy your wallet address or share the QR code with the sender.
6. The sender can then use your wallet address or scan the QR code to send you the desired asset.
```

File: src/polkaswap-staking.md [hunk 29]
```diff
@@ -0,0 +1,24 @@
...
+
+4.  After reviewing the details, click on the "Start staking" button.
+
+    Enter the amount of tokens you would like to stake or click "MAX" to stake all the available balance of the token.
+
+5.  Review your stake amount, APR, and fee
+
+![](.gitbook/assets/polkaswap-staking-review-screen.png)
+
+6.  Click "Confirm" and sign the transaction in your wallet extension.
```
Comment (outoftardis) [inline src/polkaswap-staking.md]:
```suggestion
1. Navigate to the ["Staking" page](https://polkaswap.io/#/staking/demeter/) on Polkaswap.

![](.gitbook/assets/polkaswap-staking-tab.png)

2. Click on the drop-down menu to select the preferred token you would like to stake.
3. Review all the staking details regarding this token, including the APR, total liquidity locked, reward token, and fee.

![](.gitbook/assets/polkaswap-staking-calculator.png)

4. After reviewing the details, click on the "Start staking" button.

5. Enter the amount of tokens you would like to stake or click "MAX" to stake all the available balance of the token.

6. Review your stake amount, APR, and fee.

![](.gitbook/assets/polkaswap-staking-review-screen.png)

7. Click "Confirm" and sign the transaction in your wallet extension.
```

File: src/polkaswap-connect-wallet.md (hunk 21 above)
Comment (bgodlin) [reply src/polkaswap-connect-wallet.md]: No idea, fixed now

File: src/nominating-validators.md [hunk 30]
```diff
@@ -0,0 +1,67 @@
...
+
+SORA Network uses the [Phragmèn Method](https://wiki.polkadot.network/docs/en/learn-phragmen) to select Validators.
+
+During each era, Validators are chosen using the Phragmen Method, and those who nominated them will receive a reward (a stake reward).
+
+From a Nominator's perspective, you only need to nominate at least one of the Validators who is actually elected during each era. This explains why you can nominate more than one candidate at the same time, with a maximum of 16 validators per nominator account.
+
+::: info Disclaimer
+
+It is important to nominate trustworthy and reliable validators. Validators and their Nominators' stake can be slashed, which is a punishment for failing to follow the protocol. You can read more about Slashing\* [here](https://w3f-research.readthedocs.io/en/latest/polkadot/slashing/amounts.html)
```
Comment (outoftardis) [inline src/nominating-validators.md]:
```suggestion
It is important to nominate trustworthy and reliable validators. Validators and their Nominators' stake can be slashed, which is a punishment for failing to follow the protocol. You can read more about Slashing\* [here](https://w3f-research.readthedocs.io/en/latest/polkadot/slashing/amounts.html).
```

File: src/nominating-validators.md [hunk 31]
```diff
@@ -0,0 +1,67 @@
...
+
+:::
+
+![](.gitbook/assets/polkadotjs-setup-nominator-step2.png)
+
+6. When you have selected the validators, click on "Bond & Nominate", "Sign and Submit" and then sign the transaction with Polkadot.js extension, as shown in the pictures below
```
Comment (outoftardis) [inline src/nominating-validators.md]:
```suggestion
6. When you have selected the validators, click on "Bond & Nominate", "Sign and Submit" and then sign the transaction with Polkadot.js extension, as shown in the pictures below:
```

File: src/polkaswap-other-features.md [hunk 32]
```diff
@@ -0,0 +1,9 @@
+# Other Features
+
+In the top right corner of the Polkaswap page, you'll find a few additional features that can enhance your experience.
+
+These features include the ability to switch to dark or light mode, select a language from a list of available options, enable browser notifications for balance changes and other updates, as well as hide your balances for privacy. \
```
Comment (outoftardis) [inline src/polkaswap-other-features.md]:
```suggestion
These features include the ability to switch to dark or light mode, select a language from a list of available options, enable browser notifications for balance changes and other updates, as well as hide your balances for privacy.
```

**Review by outoftardis — APPROVED** [review]

**Pull request merged**